src/test/unit/io/stan_csv_reader_test.cpp

```diff
@@ -64,7 +65,7 @@ TEST_F(StanIoStanCsvReader,read_metadata1) {
   EXPECT_FALSE(metadata.save_warmup);
   EXPECT_EQ(4085885484U, metadata.seed);
   EXPECT_FALSE(metadata.random_seed);
-  EXPECT_EQ(1U, metadata.chain_id);
+  EXPECT_EQ(0U, metadata.chain_id);
```

Comment:
This was another bug that I found while fixing this. It turned out we weren't reading the `chain_id` properly from the header info.